drivers/net/wireless/mediatek/mt76/mt76_connac_mcu.c

```diff
@@ -837,10 +837,13 @@ int mt76_connac_mcu_add_sta_cmd(struct mt76_phy *phy,
 	if (IS_ERR(skb))
 		return PTR_ERR(skb);
 
-	mt76_connac_mcu_sta_basic_tlv(skb, info->vif, info->sta, info->enable);
```

Comment:
@objelf this is even used in __mt7615_mcu_add_sta(). Can you please check if this change is fine for 7663?

Reply:
Deren mentioned the cmd  usage of mt7921 should be aligned to mt7663